packages/postgres-database/src/simcore_postgres_database/utils_users.py

```diff
@@ -134,8 +134,8 @@ async def join_and_update_from_pre_registration_details(
                 )
 
     @staticmethod
-    async def get_billing_details(conn: SAConnection, user_id: int) -> RowProxy | None:
-        result = await conn.execute(
+    def get_billing_details_query(user_id: int):
```

Comment:
why is this method public?
if this is only used below, I am not sure this brings so much clarity actually.
I actually stilll find a bit annoying to specify all single fields when one takes them all anyway. but that is webserver style now.

Reply:
I agree with you. 
In this round, I have been just gathering bits and pieces of the functions in the `users` domain.
I already did harmonize a big part of it so I would like to close this PR and do that in a different round. 
I will take note of the remaining issues in a single gh issue.